ui/locales/app.de.json

```
@@ -9,8 +9,10 @@
   "cam_lock_info_pitch": "Im Moment kann im 3D Viewer nur der Neigungswinkel verändert werden.",
   "camera_position_angle_label": "Azimut",
   "camera_position_coordinates_label": "Koordinaten",
+  "camera_position_coordinates_input_error": "Ungültige Koordinaten!",
```

I don't really know what the general tone for error messages in the viewer is, but I normally prefer more informative and neutral error messages, for example:
```suggestion
  "camera_position_coordinates_input_error": "Die eingegebenen Koordinaten sind ungültig.",
```
This would also make it clear that the error stems from a user input.